e2e/api_test.go

```diff
@@ -20,36 +22,45 @@ import (
 	"github.com/mesg-foundation/engine/ext/xnet"
 	"github.com/mesg-foundation/engine/hash"
 	pb "github.com/mesg-foundation/engine/protobuf/api"
+	"github.com/mesg-foundation/engine/server/grpc/orchestrator"
 	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc"
 )
 
 type apiclient struct {
-	pb.EventClient
+	EventClient pb.EventClient
+	// TODO: to switch when e2e tests doesn't create any event
+	// EventClient     orchestrator.EventClient
```

Comment:
i will do in a another PR. Same for https://github.com/mesg-foundation/engine/pull/1771/files#diff-d7befae5ac5d72ce8456da5eae6f2500R117-R118